.circleci/config.yml

```diff
@@ -81,6 +81,14 @@ jobs:
       - run:
           name: Build Speech (C#)
           command: artman --api=speech_v1 --language=csharp --publish=noop
+      # TODO(jskeet): Remove C#/gRPC generation tests after Cshape gapic
+      # generation includes gRPC generation.
```

Comment:
That is mainly because gapic wrapper alone won't work without grpc library. For external artman users, I would envision them to use artman to generate one self-contained working library with one single command, without knowing the relation between gapic and grpc.